### PR TITLE
Added facebook authorization and basic HomePage

### PR DESCRIPTION
**Pull request by Bartosz97**

### CODE REVIEW

**Comment by Bartosz97** [comment]
Added basic HomePage, Navbar with fb authorization.

File: src/App.css [hunk 1]
```diff
@@ -1,33 +1,23 @@
...
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+html {
+  position: relative;
+  min-height: 100%;
+  background-color: #ececec;
```
Comment (jskurzynska) [inline src/App.css]: could you add variable for this color? It's the same in line 14. so you can reuse it

File: src/containers/LoginFormContainer.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      .then(() => {
+        this.props.history.push("/");
+      })
+      .catch(err => console.log(err));
+  };
+
+  render() {
+    let fbCotent;
+
+    if (this.state.isLoggedIn) {
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: you can use destructuring state, code will look better:
**const { isLoggedIn, picture, firstName} = this.state;**  below render()
and then you can use just isLoggedIn, without repeating this.state

File: src/containers/LoginFormContainer.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          style={{
+            width: "700px",
+            margin: "auto",
+            background: "#f4f4f4",
+            padding: "20px"
+          }}
+        >
+          <img src={this.state.picture} alt={this.state.firstName} />
+          <h2>Witaj mordo: {this.state.firstName} </h2>
+          Email: {this.state.email}
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: destructuring here as well

File: src/containers/LoginFormContainer.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        </div>
+      );
+    } else {
+      fbCotent = (
+        <FacebookLogin
+          appId="355726405039496"
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: it should be const

File: src/containers/LoginFormContainer.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        token: this.state.accessToken
+      })
+      .then(() => {
+        this.props.history.push("/");
+      })
+      .catch(err => console.log(err));
+  };
+
+  render() {
+    let fbCotent;
```
Comment (jskurzynska) [inline src/containers/LoginFormContainer.js]: you can just use structure like that:
const fbContent = isLoggedIn ? (code from **if** here ) : (code from **else** here) ;

**Review by jskurzynska — CHANGES_REQUESTED** [review]

File: src/components/Navbar.js [hunk 6]
```diff
@@ -5,20 +5,95 @@ import "../App.css";
...
+        <div className="collapse navbar-collapse" id="navbarSupportedContent">
+          <form className="form-inline ml-auto my-2 my-lg-0">
+            <input
+              className="form-control mr-sm-2"
+              type="search"
+              placeholder="Search"
+              aria-label="Search"
+              style={{ width: "400px" }}
+            />
+            <fieldset class="form-group  my-2 my-lg-0">
```
Comment (jskurzynska) [inline src/components/Navbar.js]: class? it should be className

File: src/components/Navbar.js [hunk 7]
```diff
@@ -5,20 +5,95 @@ import "../App.css";
...
+              </div>
+              <div class="form-check disabled">
+                <label class="form-check-label">
+                  <input
+                    type="radio"
+                    class="form-check-input"
+                    name="optionsRadios"
+                    id="optionsRadios2"
+                    value="option2"
+                    disabled=""
```
Comment (jskurzynska) [inline src/components/Navbar.js]: do you need empty string here?

File: src/components/Navbar.js [hunk 8]
```diff
@@ -5,20 +5,95 @@ import "../App.css";
...
+                  />
+                  Pracobiorcy
+                </label>
+              </div>
+            </fieldset>
+            <button class="btn btn-secondary mx-3 my-2 my-sm-2" type="submit">
```
Comment (jskurzynska) [inline src/components/Navbar.js]: same here

**Review by jskurzynska — CHANGES_REQUESTED** [review]

File: src/App.css [hunk 9]
```diff
@@ -1,33 +1,26 @@
...
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+html {
+  position: relative;
+  min-height: 100%;
+  background-color: var(--back-color)
```
Comment (jskurzynska) [inline src/App.css:12]: 🦄 😍

File: src/containers/LoginFormContainer.js (hunk 5 above)
Comment (EnjoyTheNoise) [reply src/containers/LoginFormContainer.js]: also there's actually typo in variable name, isn't there? :P

File: src/containers/LoginFormContainer.js [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+    picture: "",
+    token: ""
+  };
+
+  componentClicked = () => console.log("clicked");
+
+  responseFacebook = response => {
+    console.log(response);
+
+    this.setState({
```
Comment (EnjoyTheNoise) [inline src/containers/LoginFormContainer.js:28]: I suggest making local copy of state, modyfing it, then passing whole modified object to setState, like:
```suggestion
    let state = this.state;
    //some magic with local state
    this.setState(state);
```
This way we know explicitly what is being passed to setState and if needed, do some prior validation on state object. Same goes for every usage of setState in PR

File: src/containers/LoginFormContainer.js [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+        firstName: this.state.first_name,
+        lastName: this.state.last_name,
+        email: this.state.email,
+        picture: this.state.picture.data.url,
+        token: this.state.accessToken
+      })
+      .then(() => {
+        this.props.history.push("/");
+      })
+      .catch(err => console.log(err));
```
Comment (EnjoyTheNoise) [inline src/containers/LoginFormContainer.js:82]: @jskurzynska is there any difference between using .catch() and just handling promise reject in .then()? I used the latter usually, but I believe it's up to our agreement which one should be used.

File: src/constants.js [hunk 12]
```diff
@@ -0,0 +1,3 @@
+export const BASE_URL = "http://localhost:56265/api";
+export const API_FB_ID = "355726405039496";
+export const API_GOOGLE_ID = "522937659687-02qcr7jftt47sbdn79vpdh0nsp1d9j0r.apps.googleusercontent.com";
```
Comment (EnjoyTheNoise) [inline src/constants.js]: uhm, should these be public?

**Review by jskurzynska — APPROVED** [review]

**Review by Astenna — APPROVED** [review]
good job c:

**Pull request merged**